.editorconfig

```diff
@@ -0,0 +1,35 @@
+root = true
+
+[*]
+# TODO: This fails in too many files, needs more investigation
+# indent_size = 4
+indent_size = unset
+indent_style = space
+
+[*.{c,cpp,h}]
+# TODO: Clashes with the current clang-format
+indent_size = unset
```

Comment:
I want to set indent size consistently with clang-format; what is the problem here?

Reply:
The problem is with `google` format which arranges it like
https://github.com/spglib/spglib/blob/a05c3d4736c12900993777ec1f85ae43ac0afa27/src/spglib.c#L74-L78
I actually don't like how wide this is making it, and instead would like to change it to something else (more pythonic), probably `clang` style. We can explore other options in https://github.com/spglib/spglib/pull/581

Reply:
clang-format will do a better job of line wrapping, but an editor uses different indent size, two (me) or eight, unless specified in .editorconfig.

Reply:
The issue is

> Wrong amount of left-padding spaces(want multiple of 4)

And this is because of how google format indents it to arbitrary length (in the case above it's 35 indent). We need to change the format style so that it is more compatible with these

Reply:
Alright.